terraform/environments/core-logging/monitoring.tf

```diff
@@ -2,3 +2,44 @@ module "core_monitoring" {
   source                     = "../../modules/core-monitoring"
   pagerduty_integration_keys = local.pagerduty_integration_keys
 }
+
+provider "aws" {
```

Comment:
What purpose does this provider fill? If it's needed it ought to be created in `providers.tf` for consistency / ease of comprehension